workflow-core/api/workflow-core.api

```diff
@@ -192,8 +196,11 @@ public abstract interface class com/squareup/workflow1/Worker {
 }
 
 public final class com/squareup/workflow1/Worker$Companion {
+	public final synthetic fun create (Lkotlin/jvm/functions/Function2;)Lcom/squareup/workflow1/Worker;
 	public final fun createSideEffect (Lkotlin/jvm/functions/Function1;)Lcom/squareup/workflow1/Worker;
 	public final fun finished ()Lcom/squareup/workflow1/Worker;
+	public final synthetic fun from (Lkotlin/jvm/functions/Function1;)Lcom/squareup/workflow1/Worker;
+	public final synthetic fun fromNullable (Lkotlin/jvm/functions/Function1;)Lcom/squareup/workflow1/Worker;
```

Comment:
These three functions are inline. I wouldn't expect they'd show up here either, although that seems less incorrect than internal functions.